packages/react-dom/src/events/getEventCode.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code#Code_values
+ * @param {object} nativeEvent
+ * @returns {string} Standardized keyboard code value if defined natively
+ */
+function getEventCode(nativeEvent: KeyboardEvent): string {
+  if (__DEV__ && nativeEvent.code === undefined) {
```

Comment:
I'm not sure if our builds will properly dead-code-eliminate this branch. I'd rather split it like this:

```js
if (__DEV__) {
  warning(nativeEvent.code !== undefined, "Warning here");
}
```

Although it should probably use `warningWithoutStack()` since it is triggered within an event handler which does not have a component stack _I think_.

https://github.com/facebook/react/blob/master/packages/shared/warningWithoutStack.js